go/tools/builders/imports.go

```diff
@@ -0,0 +1,211 @@
+package main
```

Comment:
Please add a copyright notice and license. Since this is copied from x/tools, I believe it should be the notice from the origin file.